nodes/clarify_api.js

```diff
@@ -60,10 +60,10 @@ module.exports = function (RED) {
       if (this.credentials.tokenUrl === undefined) {
         switch (url.host) {
           case 'api.clfy.io': // dev api (proxy)
-            this.credentials.tokenUrl = 'https://login.clarify.clfy.io/oauth/token';
+            this.credentials.tokenUrl = 'https://api.clfy.io/login/token';
             break;
-          case 'api.clarify.us': // prod api (proxy)
-            this.credentials.tokenUrl = 'https://login.clarify.us/oauth/token';
+          case 'api.clarify.io': // prod api (proxy)
```

Comment:
Make a default case that would work for both .us and .io (we will proxy both in the beginning)?

`default:
    this.credentials.tokenUrl = `${this.credentials.apiUrl}/login/token`
`

Reply:
I think it's actually
```js
this.credentials.tokenUrl = `${this.credentials.apiUrl}/oauth/toke`
```

Reply:
Can drop the switch-case all-together, if we are able to hold this off until we get the proxy for oauth/token up and running.

Reply:
Commit is updated. I'll won't merge PR until proxy for oauth/token is up and running.